plugins/scaffolder-backend/fixtures/test-nested-template/template-1/expected_file.ts

```diff
@@ -13,22 +13,4 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import { getRootLogger } from '@backstage/backend-common';
-import { startStandaloneServer } from './service/standaloneServer';
-
-const port = process.env.PLUGIN_PORT ? Number(process.env.PLUGIN_PORT) : 3004;
-const enableCors = process.env.PLUGIN_CORS
-  ? Boolean(process.env.PLUGIN_CORS)
-  : false;
-const logger = getRootLogger();
-
-startStandaloneServer({ port, enableCors, logger }).catch((err) => {
-  logger.error(err);
-  process.exit(1);
-});
-
-process.on('SIGINT', () => {
-  logger.info('CTRL+C pressed; exiting.');
-  process.exit(0);
-});
+console.warn('here!');
```

Comment:
Intended or debugging leftover?

Reply:
It's part of the template, maybe we could assert that the file contains `console.warn('here')` too in the tests? same for the next comment